examples/cpp/custom_payload_pool/PublisherApp.hpp

```diff
@@ -17,8 +17,8 @@
  *
  */
 
-#ifndef _FASTDDS_CUSTOM_PAYLOAD_POOL_PUBLISHER_HPP_
-#define _FASTDDS_CUSTOM_PAYLOAD_POOL_PUBLISHER_HPP_
+#ifndef FASTDDS_EXAMPLES_CPP_CUSTOM_PAYLOAD_POOL__PUBLISHER_APP_HPP
```

Comment:
Pls apply for the rest of the file
```suggestion
#ifndef FASTDDS_EXAMPLES_CPP_CUSTOM_PAYLOAD_POOL__PUBLISHERAPP_HPP
```